julia/LibCEED.jl/test/rundevtests.jl

```diff
@@ -0,0 +1,33 @@
+using Test, LibCEED, LinearAlgebra, StaticArrays
+
+@testset "LibCEED Development Tests" begin
+    @test ceedversion() isa VersionNumber
+    @test isrelease() == false
+
+    @testset "CeedVector" begin
+        n = 10
+        c = Ceed()
+
+        v1 = rand(n)
+        v2 = rand(n)
+        v3 = rand(n)
+
+        cv1 = CeedVector(c, v1)
+        cv2 = CeedVector(c, v2)
+        cv3 = CeedVector(c, v3)
+
+        alpha = rand()
+
+        scale!(cv1, alpha)
+        v1 .*= alpha
```

Comment:
I think Julia will let us overload .*= if we want

Reply:
It's actually not so easy to properly overload the broadcast operators, unfortunately.

It is possible to do, but there are some design decisions to be made, and it will likely have some limitations. I think keeping functions like `scale!` and `axpy!` explicit is a better choice, because it will also prevent users from writing accidentally slow code.